package.json

```diff
@@ -33,13 +33,16 @@
   },
   "homepage": "https://github.com/blittle/single-spa-vuejs#readme",
   "devDependencies": {
-    "@babel/cli": "^7.5.5",
     "@babel/core": "^7.5.5",
-    "@babel/plugin-transform-modules-umd": "^7.2.0",
     "@babel/preset-env": "^7.5.5",
+    "@rollup/plugin-commonjs": "^11.0.1",
+    "@rollup/plugin-node-resolve": "^7.0.0",
     "@types/jest": "^24.0.15",
     "babel-jest": "^24.8.0",
+    "css.escape": "^1.5.1",
```

Comment:
:nerd glasses on: technically a normal dependency and not a devDep right? 

not a big deal though haha :)

Reply:
Intentionally a devDependency - since it's bundled into the lib directory, we don't need users of single-spa-vue to download it separately into node_modules.